### ISSUE TITLE
Rivedere la progettazione delle pagine di gestione

### ISSUE THREAD

**Issue by rzaupa** [Gruppo-TecWeb/Codice#296]
Mi chiedevo se avesse senso aprire la stessa pagina ("gestione classifica") che cambia (solo nel contenuto) a seconda di che pulsante clicchi ("modifica" o "aggiunta") quando dal punto di vista teorico dovrebbero essere "evidentemente" due pagine differenti, per breadcrumb e per titolo. Questo riguarda ovviamente non solo classifiche ma anche le altre pagine. Questione non banale e da non sottovalutare.... ad esempio vedi https://web.math.unipd.it/accessibility/dettaglio.html?ID=3

Questo esempio si applica anche a tutti gli altri tipi di gestione come tipi eventi , eventi, rappers e amministratori

**Comment by abenetazzo** [comment]
> ... ad esempio vedi https://web.math.unipd.it/accessibility/dettaglio.html?ID=3

L'esempio riportato non mi sembra pertinente, in quanto il codice sorgente della pagina che viene consegnata al browser ha il tag title correttamente valorizzato all'interno dell'header.

Attualmente non mi sembra ci sia nulla di sbagliato in titolo e breadcrumbs in quanto sia nel caso di un'aggiunta che nel caso di una modifica il termine gestione ben si adatta. Non mi sembra sensato sdoppiare una pagina, visto che allo stato attuale i contenuti sono esattamente i medesimi (si tratta dello stesso form), separare questo contenuto in due pagine complicherebbe solo la futura manutenzione del codice e potrebbe anche creare confusione nell'utente che aprendo due pagine distinte (url diversi) si ritroverebbe praticamente gli stessi contenuti al netto di alcune singole parole.

Ad ogni modo penso sarebbe cosa buona ed utile confrontarci e valutare la cosa, sono sicuro che tutta l'area admin si potrebbe migliorare, ma visti i tempi ristretti e visto che non mi sembra ci sia alcun problema dal punto di vista del progetto da consegnare tra poco più di una settimana direi di rimandare questa issue a dopo la consegna del progetto.

**Comment by rzaupa** [comment]
> > ... ad esempio vedi https://web.math.unipd.it/accessibility/dettaglio.html?ID=3
> 
> L'esempio riportato non mi sembra pertinente, in quanto il codice sorgente della pagina che viene consegnata al browser ha il tag title correttamente valorizzato all'interno dell'header.
> 
> Attualmente non mi sembra ci sia nulla di sbagliato in titolo e breadcrumbs in quanto sia nel caso di un'aggiunta che nel caso di una modifica il termine gestione ben si adatta. Non mi sembra sensato sdoppiare una pagina, visto che allo stato attuale i contenuti sono esattamente i medesimi (si tratta dello stesso form), separare questo contenuto in due pagine complicherebbe solo la futura manutenzione del codice e potrebbe anche creare confusione nell'utente che aprendo due pagine distinte (url diversi) si ritroverebbe praticamente gli stessi contenuti al netto di alcune singole parole.
> 
> Ad ogni modo penso sarebbe cosa buona ed utile confrontarci e valutare la cosa, sono sicuro che tutta l'area admin si potrebbe migliorare, ma visti i tempi ristretti e visto che non mi sembra ci sia alcun problema dal punto di vista del progetto da consegnare tra poco più di una settimana direi di rimandare questa issue a dopo la consegna del progetto.

Certo, non ho mai detto di fare due file html, ho detto solo che il title e le breadcrumb si potevano adattare a seconda della pagina dato che così almeno lato utente è piu' evidente che siano pagine diverse. Non ho mai detto di ristrutturare tutta la parte admin ma si tratterebbe credo di una modifica minore lato php. Però concordo che altre questioni hanno piu' priorità.
